pallets/restricted-tokens/src/lib.rs

```diff
@@ -493,49 +492,22 @@ pub mod pallet {
 			who: <T::Lookup as StaticLookup>::Source,
 			currency_id: T::CurrencyId,
 			#[pallet::compact] new_free: T::Balance,
-			#[pallet::compact] new_reserved: T::Balance,
```

Comment:
Why this: https://kflabs.slack.com/archives/C05TBFRBL15/p1715597422917969

Open to discussion

Reply:
Finally field restored using `HoldReason::NativeIndex`: [38377a9](https://github.com/centrifuge/centrifuge-chain/pull/1833/commits/38377a91aa8cc6c25f650ef9d975294579eae437)